### ISSUE TITLE
Latest update for native wakeLock results in build errors during SSR

### ISSUE THREAD

**Issue by canibanoglu** [richtr/NoSleep.js#96]
* NoSleep.js Version: 0.11.0

### Expected Behavior
For builds to not fail due `navigator` during SSR.

### Actual Behavior
This happened to me with a `gatsby` project but technically this can happen to any project that has SSR. 

This is almost certainly due to https://github.com/richtr/NoSleep.js/pull/95/files#diff-b4c19cbd92d000bcb4eedad9beb3fb70R118 . I believe that code should be wrapped in an if block so that it doesn't throw during server side rendering.

### Code
I'm getting the build error due to the following line: 
https://github.com/canibanoglu/teacompanion/blob/master/src/components/Countdown/index.jsx#L2

The error: 

```
"navigator" is not available during server side rendering.

See our docs page for more info on this error: https://gatsby.dev/debug-html

  WebpackError: ReferenceError: navigator is not defined

```

Happy to create a PR once someone confirms

### How Do We Reproduce?
You can clone https://github.com/canibanoglu/teacompanion, change `nosleep.js` version in `package.json` to `0.11.0` and run `yarn build`. It will fail with the above message.

**Comment by ReinoutStevens** [comment]
Confirming that I am getting similar errors using NextJS

**Comment by richtr** [comment]
Fixed by 2852d1550e1081dcf4821b067087bce503afebfc!
